specification.md

```diff
@@ -1655,22 +1656,22 @@ The Agent SHOULD periodically report its metrics to the destination offered in t
 interval is 10 seconds. Here is the diagram that shows the operation sequence:
 
 ```
-                Agent                          Server
+       Agent    Client                                Server
                                                             Metric
-                  │                                      │  Backend
-                  │ServerToAgent{ConnectionSettingsOffer}│
-                  │◄─────────────────────────────────────┤    │
-                  │                                      │    │
-                  │                                           │
+        │         │                                      │  Backend
+        │         │ServerToAgent{ConnectionSettingsOffer}│
+        ┌─────────│◄─────────────────────────────────────┤    │
+        │         │                                      │    │
+        ▼         │                                           │
```

Comment:
I think we can delete the vertical line for Client on lines 1665-1676. This will eliminate the intersection with horizontal arrows on lines 1668 and 1674.

Reply:
Yes, it will look better!